test/karma/test-app/components.d.ts

```diff
@@ -1490,7 +1498,11 @@ declare namespace LocalJSX {
     }
     interface ShadowDomBasicRoot {
     }
-    interface ShadowDomMode {
+    /**
+   * 
+   * @virtualProp {string} colormode - The mode determines which platform styles to use.
+   */
+  interface ShadowDomMode {
```

Comment:
I noticed this indentation change here, do you think we can fix this so that it's more like this?

```suggestion
    /**
     * 
     * @virtualProp {string} colormode - The mode determines which platform styles to use.
     */
    interface ShadowDomMode {
```

Reply:
a79f1a3